### ISSUE TITLE
fs mount command show all mount point as ufs type local after master restart

### ISSUE THREAD

**Issue by LuQQiu** [Alluxio/alluxio#8825]
**Alluxio Version:**
2.0.0-preview

**Describe the bug**
```
// Start alluxio
$./bin/alluxio-start.sh all SudoMount
// Mount s3 to alluxio
$./bin/alluxio fs mount --option aws.accessKeyId=**** --option aws.secretKey=*** /s3-mnt s3a://alluxio-test/
// fs mount show the right ufs type s3
$./bin/alluxio fs mount
s3a://alluxio-test-fuse/                             on  /s3-mnt  (s3, capacity=-1B, used=-1B, not read-only, not shared, properties={aws.secretKey=*****, aws.accessKeyId=*****})
/Users/alluxio/alluxioFolder/alluxio/underFSStorage  on  /        (local, capacity=464.77GB, used=-1B(0%), not read-only, not shared, properties={})

// restart alluxio
$./bin/alluxio-start.sh all SudoMount

// All mount point ufs type is local now
$./bin/alluxio fs mount
s3a://alluxio-test-fuse/                             on  /s3-mnt  (local, capacity=-1B, used=-1B, not read-only, not shared, properties={aws.secretKey=*****, aws.accessKeyId=*****})
/Users/alluxio/alluxioFolder/alluxio/underFSStorage  on  /        (local, capacity=464.77GB, used=-1B(0%), not read-only, not shared, properties={})

```

**Expected behavior**
Ufs type should always show the right value across master restart

**Comment by aaudiber** [comment]
Does this only affect how the CLI displays info, or could there be other impact on the mount point?

**Comment by LuQQiu** [comment]
It affects one of the future PR which is about hiding the aws (or other object stores) secret keys.
I will take a look into this issue when I am working on that PR.

**Comment by apc999** [comment]
@LuQQiu has this issue been fully addressed by PR https://github.com/Alluxio/alluxio/pull/8885? If yes, you can close this Issue

**Comment by apc999** [comment]
also is this issue also in 1.8? if yes, can you cherry-pick your fix to 1.8 branch?

**Comment by LuQQiu** [comment]
I checked that branch 1.8 do not have this issue. It's a newly occurred issue. Fixed in #8885
